var/spack/repos/builtin/packages/r-pinfsc50/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RPinfsc50(RPackage):
+    """Sequence ('FASTA'), Annotation ('GFF') and Variants ('VCF') for 17
+    Samples of 'P. Infestans" and 1 'P. Mirabilis'
+
+    Genomic data for the plant pathogen "Phytophthora infestans." It includes
+    a variant file ('VCF'), a sequence file ('FASTA') and an annotation file
+    ('GFF'). This package is intended to be used as example data for packages
+    that work with genomic data."""
+
+    homepage = "https://cran.r-project.org/web/packages/pinfsc50/index.html"
```

Comment:
```suggestion
    homepage = "https://cloud.r-project.org/web/packages/pinfsc50/index.html"
```
Alternatively, since that is just the CRAN canonical home page, you could remove that and the `cran` attribute will handle it.